.github/workflows/webassembly.yml

```diff
@@ -55,17 +56,19 @@ jobs:
           path: 'openvino'
           submodules: 'true'
 
-      - name: Install ccache
-        run: apt-get install --assume-yes --no-install-recommends ccache
+      - name: Install sccache
+        uses: mozilla-actions/sccache-action@v0.0.3
+        with:
+          version: "v0.5.4"
 
       - name: emcmake cmake - configure
         run: |
           emcmake cmake \
             -DCMAKE_CXX_FLAGS="-Wno-deprecated-declarations" \
             -DCMAKE_C_FLAGS="-Wno-deprecated-declarations" \
             -DCMAKE_COMPILE_WARNING_AS_ERROR=ON \
-            -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
-            -DCMAKE_C_COMPILER_LAUNCHER=ccache \
+            -DCMAKE_CXX_COMPILER_LAUNCHER=${{ env.CMAKE_CXX_COMPILER_LAUNCHER }} \
+            -DCMAKE_C_COMPILER_LAUNCHER=${{ env.CMAKE_C_COMPILER_LAUNCHER }} \
```

Comment:
let's align env usage in bash scripts: ${{ env.CMAKE_CXX_COMPILER_LAUNCHER }} -> ${CMAKE_CXX_COMPILER_LAUNCHER}

Reply:
we use `${}` only for paths, others still use `${{ env.XXX }}`

Reply:
It'll be better to use `${FOOBAR}` syntax for all environment variables just for the sake of uniformity, but this is out of the scope of current PR.

Reply:
just for the future, what is the reason? 
This is bash command and it more native to use it's syntax